### PR TITLE
Create Portuguese-BR

### PR DESCRIPTION
**Pull request by leonardo403**
Check out this file.

### CODE REVIEW

**Comment by phanan** [comment]
Thanks! Please use a `.md` extension.

**Comment by leonardo403** [comment]
Ok,
I will change the file and send again.

File: Translations/Portuguese-BR/README.md [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+## Créditos
+O que estamos fazendo aqui é principalmente a coleta trechos úteis de todo o interwebs (por exemplo, uma boa parte é de
+[Apache Server Configs](https://github.com/h5bp/server-configs-apache)) em um só lugar. Enquanto estamos tentando dar crédito onde, devido, as coisas podem estar em falta. Se você acreditar em qualquer coisa que está aqui deve ser dado o seu trabalho e créditos, avise-nos, ou apenas enviar uma PR.
+
+## Tabela de Conteúdo
+- [Rewrite and Redirection](#rewrite-and-redirection)
+    - [Force www](#force-www)
+    - [Force www in a Generic Way](#force-www-in-a-generic-way)
+    - [Force non-www](#force-non-www)
+    - [Force non-www in a Generic Way](#force-non-www-in-a-generic-way)
```
Comment (phanan) [inline Translations/Portuguese-BR/README.md]: Broken link

**Comment by phanan** [comment]
Awesome. Can you fix the broken link and squash all commits into one?

**Comment by phanan** [comment]
I messed up something with your PR, sorry.

**Pull request merged**